zone-service/build.gradle

```diff
@@ -16,12 +16,25 @@ dependencies {
     }
     implementation(project(":cloudfoundry-identity-server"))
     implementation(project(":cloudfoundry-identity-model"))
+    implementation(project(":cloudfoundry-identity-metrics-data"))
     implementation(libraries.hibernateValidator)
     implementation(libraries.slf4jImpl)
     implementation(libraries.log4jCore)
+    implementation(libraries.bouncyCastleProv)
+    implementation(libraries.bouncyCastlePkix)
+    implementation(libraries.springSecurityOauth) {
+        exclude(module: "commons-codec")
+        exclude(module: "jackson-mapper-asl")
+        exclude(module: "spring-security-web")
+    }
+    implementation 'org.apache.commons:commons-lang3:3.0'
     testImplementation(libraries.springBootStarterTest)
 }
 
 test {
     useJUnitPlatform()
+}
+
+tasks.withType(AbstractCompile) {
```

Comment:
Hence, sun.security.x509 packages in JDK 11, added to gradle compiler args :  options.compilerArgs += ["--add-exports", "java.base/sun.security.provider=ALL-UNNAMED"]

[Access `sun.security.x509` in JDK 11 without modules?](https://stackoverflow.com/questions/61149455/access-sun-security-x509-in-jdk-11-without-modules).

Reply:
The link you are referring to also mentions that this is just a workaround and "is liable to stop working in the future. The solution would be to find a way to avoid depending on those classes."

You might want to explore [java.security.cert](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/security/cert/package-summary.html) package or [bouncy-castle](https://stackoverflow.com/questions/29852290/self-signed-x509-certificate-with-bouncy-castle-in-java)

We could address it as a separate story.

Reply:
hence there are only two string constants are required from sun.security.provider.X509Factory.BEGIN_CERT
sun.security.provider.X509Factory.END_CERT, changed as declared constants for the same and removed the import sun.security.provider.X509Factory.
I have excluded gradle compiler args from build.gradle file.

Done. Thanks!